lib/publiccloud/ec2.pm

```diff
@@ -184,9 +184,8 @@ sub img_proof {
 
 sub cleanup {
     my ($self, $args) = @_;
-    #if for some reason the instance is not created everything below does not make sense to execute
-    return unless ($args->{my_instance} && $args->{my_instance}->{instance_id});
-    my $instance_id = $args->{my_instance}->{instance_id};
+    script_run('cd ~/terraform');
```

Comment:
This leaves the default console in `~/terraform` for all following tests. You perhaps want to run a `script_run("cd") after the following `script_output`.

While every test module should ensure to be in the right directory, most blindly assume to be in `~`, so this has the potential for unintended side-effects we probably want to mitigate.